stubs/urllib3/urllib3/connectionpool.pyi

```diff
@@ -44,7 +44,7 @@ log: Logger
 
 class ConnectionPool:
     scheme: ClassVar[str | None]
-    QueueCls: ClassVar[Queue[Any]]
+    QueueCls: ClassVar[LifoQueue]
```

Comment:
No, stubtest was complaining because you had `ClassVar[Queue[Any]]`, when you should have had `ClassVar[type[Queue[Any]]]`

Reply:
That should be fixed in the latest revision

Reply:
It's not, because (putting aside the disagreement over which queue class it should be), you're saying it's an instance of a class, but it's not, it's the class itself. Even if you want `LifoQueue`, it needs to be `ClassVar[type[LifoQueue]]`, not `ClassVar[LifoQueue]`

Reply:
^You made that update.

Reply:
Just a miscommunication caused by using async comments :)

Reply:
Race conditions! Everywhere I look! More race conditions! :)